README.md

```diff
@@ -1,5 +1,7 @@
 # HR-TIL
 
+![til](https://cloud.githubusercontent.com/assets/11463275/26385291/8125cf72-400c-11e7-8779-1fdb802c9d6f.png)
```

Comment:
sure, the image was on a clone I made of the project instead of the fork. Where would you like me to put the image? Is there an issue on github where you store the images?